core/engine/src/builtins/array_buffer/mod.rs

```diff
@@ -554,7 +585,7 @@ impl ArrayBuffer {
     /// [`ArrayBuffer.prototype.resize ( newLength )`][spec].
     ///
     /// [spec]: https://tc39.es/ecma262/#sec-arraybuffer.prototype.resize
-    pub(crate) fn resize(
+    pub(crate) fn js_resize(
```

Comment:
Why not call `resize` from here? It would remove a lot of duplication.

Reply:
I didn't want to change the behaviour of JS code.